tests/test_polyMap.py

```diff
@@ -179,6 +223,42 @@ def test_PolyMapPolyTran(self):
         pin2 = pm2.tranInverse(pout)
         npt.assert_allclose(pin, pin2, atol=1.0e-10)
 
+    def test_PolyMapPolyMapUnivertible(self):
+        """Test polyTran on a PolyMap without a single-valued inverse
+
+        The equation is y = x^2 - x^3, whose inverse has 3 values
+        between roughly -0.66 and 2.0
+        """
+        coeff_f = np.array([
+            [2.0, 1, 2],
+            [-1.0, 1, 3],
+        ])
+        pm = astshim.PolyMap(coeff_f, 1, "IterInverse=1")
+
+        self.checkBasicSimplify(pm)
+        self.checkCopy(pm)
+        self.checkPersistence(pm)
+
+        pin = np.array([
+            [-0.5],
+            [0.5],
+            [1.1],
+            [1.8],
+        ])
+        des_pout = 2.0*pin**2 - pin**3
+        pout = pm.tranForward(pin)
+        npt.assert_allclose(pout, des_pout)
```

Comment:
I agree that `pin` is not a great name; `indata` or `inarr` would probably be better. However, fixing that is out of scope for this ticket. In not keen to use `x / y` since it breaks down for large numbers of axes (which doc string refers to those names?).

`des` is short for `desired`.

A good time to fix the names is when I stop transposing data in astshim, since I will have to modify the input and expected output data for most of the unit tests.